experimental/dds/sequence-deprecated/src/test/tsconfig.json

```diff
@@ -0,0 +1,21 @@
+{
+    "extends": "@fluidframework/build-common/ts-common-config.json",
+    "compilerOptions": {
```

Comment:
FYI: #10957 turns on strict null checks in sequence. Dunno which will merge first, but eventually we should have it on here as well.

Reply:
well, at least in the non-test config :)